### PR TITLE
Centre map layout, colour enhancement for UX

### LINKED ISSUE #6365: Map content should be centered on full-screen view
The column of content on the /map view should be centered on the page when viewed full screen.

## Mockup
image

I don't think it needs to be centered on the side view, unless centering doesn't interfere with the display.

Ref: https://www.reddit.com/r/FreeCodeCamp/comments/41vr0i/please_bring_back_brownish_fcc_old_version/cz73beo

### PR DESCRIPTION
**Pull request by justincorrigible**
Centring in screen, responsive and indented.

Additional styling to enhance visual hierarchies, and then some UX colour-loving care to ease use on the sidebar+menu.

---

Tested locally and on mobile (Chrome for iOS)
Closes #6365 ;)

---

screen shot 2016-01-23 at 23 11 47

screen shot 2016-01-24 at 13 53 47

### CODE REVIEW

File: client/less/map.less [hunk 1]
```diff
@@ -73,27 +82,100 @@
...
   top: 0;
 }
+.map-fixed-header p {
+  margin: 10px 0 20px;
+}
+.map-fixed-header hr {
+  margin:30px 0;
+}
+.map-buttons button {
+  width:300px;
```
Comment (bugron) [inline client/less/map.less]: Do we really need a fixed width button?

**Comment by bugron** [comment]
I don't like grayed backgrounds for navbar links. Lots of changes to center the menu. I'd prefer this solution https://github.com/FreeCodeCamp/FreeCodeCamp/pull/6373.

**Comment by SaintPeter** [comment]
@bugron I worked with @hallaathrad on this one - I think it's a general UX improvement, not just the centering.  The centering was the start, but I live the improved look and feel.

**Comment by bugron** [comment]
Why is  better than  ?

File: client/less/map.less (hunk 1 above)
Comment (hallaathrad) [inline client/less/map.less]: Those buttons extend to whatever container they are in... courtesy of bootstrap.
There's absolutely no use or aesthetic gain to having those buttons changing size.

**Comment by hallaathrad** [comment]
@bugron Thanks for all your feedback! Let me see if I can address your concerns:

Does not having a background to divide the sections work better than having one (grey in this case) at specifying clear visual hierarchies? We are building this for our users and their ease. 

screen shot 2016-01-24 at 08 32 00
The reason I changed the colour of the options in the navbar is because I wanted to make it obvious the map is what is being displayed. It is meant as a visual connection, which I'm still working on. Think of a tab, of sorts. In fact, I've just realised that can be an `active` status, rather than a hover... 
Gotta fix the hover on those other green buttons anyway...
I'm taking that navbar change right now. Amending this commit :) thanks for the idea.

And I agree, the space between them the character blocks is indeed somewhat big, but it's due to the massive size of those fonts —Which I didn't change, as I was not sure how much of the original design to modify freely. However, any less "air" between those elements as they are would make them look cramped and not considered in the design. 

screen shot 2016-01-24 at 09 10 41

vs.

screen shot 2016-01-24 at 08 35 26 
---------
This would be in the next commits with other enhancements @SaintPeter and I have in mind... notice how smaller text, allows for less space between the elements.

In general, given Quincy's directive of following Quora's minimalist approach, white space becomes the single most starring design element. It is quite common seeing people being afraid of it too. 
White space is always critical in layout composition... which Quora themselves don't get quite right IMO. But WE will. (though it'll take a few steps haha). 

Following the agile principles, I've committed quickly, and intend on doing it very often ;) so please bear with me and my ideas as I slowly build upon them.

**Comment by hallaathrad** [comment]
I've amended with some of @bugron 's feedback on spacing and colouring. Please pull and check again?

**Comment by hallaathrad** [comment]
Methinks this is good to go. anyone care to give it a try? It's not too too late to make further adjustments or include requests :)

**Comment by QuincyLarson** [comment]
@bugron Actually, the on-hover color used to be brown when the nav was brown. I agree that darkgreen is better here.

**Comment by QuincyLarson** [comment]
@hallaathrad this looks much better! My only suggestion is that we remove the gray top bar and shift things back up like we had them.
1. We want as much vertical real estate as possible to display the map below the HR
2. We want the map to look more like the chat drawer, which doesn't have a gray bar.
I realize the purpose of this is to draw attention to the close and pop-out button, but I think it's not worth this tradeoff.

Overall, the typography looks much better, and though it's less compact, I think the gray stripes make it much easier to read, and having the time on its own line looks better. I can't articulate why, but it does.

**Comment by hallaathrad** [comment]
Amended: I've removed the grey background for the actions, as @QuincyLarson requested. 
If nobody else has any further feedback, I believe this one is (finally) good to go :+1:

**Comment by SaintPeter** [comment]
Looking good to me.

@QuincyLarson - I'd like your sign-off since it's a UI/UX change.

image

**Comment by hallaathrad** [comment]
Added a simple detail I just noticed and thought would be nice... up for feedback and maybe merging?
screen shot 2016-01-26 at 08 52 45

**Comment by AkiraLaine** [comment]
This looks good! Tested locally, should I merge it?

**Comment by SaintPeter** [comment]
I think we're just waiting on Quincy to give the final word.

**Comment by AkiraLaine** [comment]
Oh ok.

**Comment by QuincyLarson** [comment]
@hallaathrad Actually, I am not sure that the bright green with grayed out text look good together. I had it this way previously and a camper (who was ostensibly a designer) said it was an antipattern.

That's why I switched both completed challenges (and their checkmarks) to being faded.

Also, using gray text makes these not look like links anymore, and they are still links.

**Comment by hallaathrad** [comment]
Hmm yes, I can see now why they would've seen it as such. How about a darker green then?. My reasoning was that the current fading didn't make it different enough to give that feeling of "done and over".

**Comment by QuincyLarson** [comment]
Awesome. Yes, I agree. If you think this faded green is still sufficiently legible, it looks much better and conveys the notion of "past" or "completed" much better.

**Pull request merged**